src/utils/colors/formatters.ts

```diff
@@ -29,12 +29,10 @@ export const convert255ScaleRGBtoDecimal = (color: {
   return { alpha: (alpha ?? 255) / 255, b: b / 255, g: g / 255, r: r / 255 };
 };
 
-export const formatForOklchDisplay = (oklch: Oklch): string => {
-  return `${toPercent(oklch.l)} ${clearValue(oklch.c)} ${clearValue(
+export const formatForOklchDisplay = (oklch: Oklch): string => `${toPercent(oklch.l)} ${clearValue(oklch.c)} ${clearValue(
```

Comment:
Jfyi, looks a bit strange to mix preact migration with style fixes not related to the main issue

Reply:
i guess it happened when i changed eslint react to preact